README.rst

```diff
@@ -164,7 +164,7 @@ To install:
 .. code-block:: bash
 
    cd giotto-tda
-   pip install -e ".[tests, doc]"
+   pip install -e ".[dev]"
```

Comment:
These are just README instructions but do you and @rth think we should sell the `python -m pip install` idea here too? What would be the disadvantages in doing that?